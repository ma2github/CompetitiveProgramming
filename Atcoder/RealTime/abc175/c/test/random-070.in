-556618801662711 955386642912743 914773798029738
