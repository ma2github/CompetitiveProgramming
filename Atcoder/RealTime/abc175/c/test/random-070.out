358154996367027
